### DISCUSSION TITLE
JKBMS alarms questions

### DISCUSSION THREAD

**Discussion by seamaster101** [Louisvdw/dbus-serialbattery#614] [Q&A]
I have quite few installations with JKBMSs. I would like to make sure all alarms work well. Currently i have setup on the bench 2 batteries 4cells each with separate BMSs.
I see them in Venus OS and some parameters are shown, some are not.
I ran dbus-spy and get the information that is received from the BMS and maybe some  that is set by the configuration file utils.py

<img width="820" alt="image" src="https://user-images.githubusercontent.com/7897023/236600024-1be3e53e-d85b-4127-bc7c-c7d15bdd3a7e.png">

My first question is why we don't see Alarms/NighCellVoltage (represented wth "-") and ever other alarm appears properly with "0" (i presume "0" is not triggered and "1" is triggered?
  
My second question is how can i trick the systems to show me some alarms. Do i edit the "utils.py" file temporary with some artificial values that trigger alarms?

I will greatly appreciate some guidance!
thank you in advance

**Comment by mr-manuel** [comment]
https://github.com/Louisvdw/dbus-serialbattery/blob/aff297ecb260da7f085d03846c43adf09292aeaf/etc/dbus-serialbattery/dbushelper.py#L262

Just press enter on the dbus-spy and enter the value 1. Then you see for a second the alarm.
0 = OK
1 = Warning
2 = Critical
